docs/autogen.py

```diff
@@ -99,6 +99,7 @@
     'deserialize',
     'get',
     'set_image_dim_ordering',
+    'normalize_data_format',
```

Comment:
It's not part of the public API, it shouldn't be documented.

Reply:
Indeed. This is why it's in the `EXCLUDE` set here.